web/components/GigInfo/GigInfo.js

```diff
@@ -62,41 +70,50 @@ export function GigInfo({ gigData }) {
                     <li className="md:pr-3 md:mr-3 lg:border-r lg:border-black pr-5 mr-5">
                       <label className="mr-1 text-gray-800">Opened:</label>
                       {formatDistance(
-                        subDays(new Date(gigData.createdDate), 3),
+                        subDays(new Date(bounty.createdAt), 3),
                         new Date()
-                      )}
+                      )}{' '}
                       ago
                     </li>
                     <li className="md:pr-3 md:mr-3 lg:border-r lg:border-black lg:my-0 pr-5 my-2 mr-5">
                       <label className="mr-1 text-gray-800">Gig Type:</label>
-                      {gigData.type}
+                      {bounty.categories &&
```

Comment:
I believe categories should always be defined

Reply:
fixed